CHANGELOG.md

```diff
@@ -67,6 +67,14 @@ All notable changes for each version of this project will be documented in this
     - `increment` and `decrement` methods now accept an optional `delta` parameter which targets the currently spun date portion. It takes precedence over the values set in `spinDelta`.
 - `IgxDatePicker`
     - **Feature** - `value` accepts ISO 8601 string format.
+    - **Feature** - The actions template now exposes the Calendar component as implicit context:
```

Comment:
@Lipata These _should really_ be in the 'New Features' section, but I'm leaving it here so everything can be moved together